libs/carrier-portal/types/src/lib/person-view-data.ts

```diff
@@ -135,6 +136,7 @@ export const defaultPolicy: Policy = {
     hios_id: '',
     name: '',
     product_kind: '',
+    coverage_year: '2022',
```

Comment:
can you update this to empty string? it's giving false sense that we're hard coding year here!